app/controllers/invite_controller.rb

```diff
@@ -0,0 +1,33 @@
+class InviteController < ApplicationController
+  def new; end
+
+  def create
+    @friend = \
+      GitHubService.new(current_user.gh_token).get_user(params[:github_handle])
```

Comment:
I think I have some ideas about how to shorten these lines and can make a PR after these two for general housecleaning